nebula_decoders/src/nebula_decoders_continental/decoders/continental_ars548_decoder.cpp

```diff
@@ -0,0 +1,574 @@
+    object_msg.absolute_velocity_std.y = static_cast<double>(object.dynamics_abs_vel_y_std.value());
+    object_msg.absolute_velocity_covariance_xy = object.dynamics_abs_vel_covariance_xy.value();
+
+    object_msg.relative_velocity.x = static_cast<double>(object.dynamics_rel_vel_x.value());
+    object_msg.relative_velocity.y = static_cast<double>(object.dynamics_rel_vel_y.value());
+    object_msg.relative_velocity_std.x = static_cast<double>(object.dynamics_rel_vel_x_std.value());
+    object_msg.relative_velocity_std.y = static_cast<double>(object.dynamics_rel_vel_y_std.value());
+    object_msg.relative_velocity_covariance_xy = object.dynamics_rel_vel_covariance_xy.value();
+
+    object_msg.absolute_acceleration.x = static_cast<double>(object.dynamics_abs_accel_x.value());
+    object_msg.absolute_acceleration.y = static_cast<double>(object.dynamics_abs_accel_y.value());
+    object_msg.absolute_acceleration_std.x =
+      static_cast<double>(object.dynamics_abs_accel_x_std.value());
+    object_msg.absolute_acceleration_std.y =
+      static_cast<double>(object.dynamics_abs_accel_y_std.value());
+    object_msg.absolute_acceleration_covariance_xy =
+      object.dynamics_abs_accel_covariance_xy.value();
+
+    object_msg.relative_velocity.x = object.dynamics_rel_accel_x.value();
+    object_msg.relative_velocity.y = object.dynamics_rel_accel_y.value();
+    object_msg.relative_velocity_std.x = object.dynamics_rel_accel_x_std.value();
+    object_msg.relative_velocity_std.y = object.dynamics_rel_accel_y_std.value();
+    object_msg.relative_velocity_covariance_xy = object.dynamics_rel_accel_covariance_xy.value();
+
+    object_msg.orientation_rate_mean = object.dynamics_orientation_rate_mean.value();
+    object_msg.orientation_rate_std = object.dynamics_orientation_rate_std.value();
+
+    object_msg.shape_length_edge_mean = object.shape_length_edge_mean.value();
+    object_msg.shape_width_edge_mean = object.shape_width_edge_mean.value();
+  }
+
+  object_list_callback_(std::move(msg_ptr));
+
+  return true;
+}
+
+bool ContinentalARS548Decoder::ParseSensorStatusPacket(
+  const std::vector<uint8_t> & data, const std_msgs::msg::Header & header)
+{
+  SensorStatusPacket sensor_status_packet;
+  std::memcpy(&sensor_status_packet, data.data(), sizeof(SensorStatusPacket));
+
+  radar_status_.timestamp_nanoseconds = sensor_status_packet.stamp.timestamp_nanoseconds.value();
+  radar_status_.timestamp_seconds = sensor_status_packet.stamp.timestamp_seconds.value();
+
+  if (sensor_status_packet.stamp.timestamp_sync_status == 1) {
+    radar_status_.timestamp_sync_status = "SYNC_OK";
+  } else if (sensor_status_packet.stamp.timestamp_sync_status == 2) {
+    radar_status_.timestamp_sync_status = "NEVER_SYNC";
+  } else if (sensor_status_packet.stamp.timestamp_sync_status == 3) {
+    radar_status_.timestamp_sync_status = "SYNC_LOST";
+  } else {
+    radar_status_.timestamp_sync_status = "INVALID_VALUE";
+  }
```

Comment:
This would be cleaner as a switch statement

Reply:
Addressed in [800b351](https://github.com/tier4/nebula/pull/123/commits/800b351ad5c941b14e31507c4cd59eb6b42ddd4a)